activerecord/lib/active_record/persistence.rb

```diff
@@ -421,20 +422,18 @@ def toggle!(attribute)
     #   assert_equal 25, account.credit        # check it is updated in memory
     #   assert_equal 25, account.reload.credit # check it is also persisted
     #
-    # Another common use case is optimistic locking handling:
+    # Another common use case is optimistic lock handling:
```

Comment:
The feature is called `optimistic locking`, so it's better to use that phrasing exactly. Should be able to with:

```ruby
# Another common use case is handling optimistic locking:
```